consumer/src/main/resources/application-event.yml

```
@@ -0,0 +1,23 @@
+      definition: comment; like; follow; # 메시지를 수신했을 때 처리할 함수 정의
+    stream:
+      binder:
+        brokers: localhost:9092
+      bindings:
+        comment-in-0: # comment 토픽에서 메시지를 수신하는 채널
+          destination: comment # 메시지를 수신할 토픽 이름
+          content-type: application/json # 직렬화 / 역직렬화
+          group: notification-consumer # 컨슈머 그룹 이름
+          consumer.max-attempts: 2 # 메시지 처리 실패 시 재시도 횟수
+        like-in-0:
+            destination: like
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 10 but found 12

(indentation)